deployment/SampleClaimsProvider/Controllers/HealthCheckController.cs

```diff
@@ -0,0 +1,21 @@
+﻿// ---------------------------------------------------------------------------------
+// <copyright company="Microsoft">
+//   Copyright (c) {Microsoft} Corporation. All rights reserved.
+// </copyright>
+// ---------------------------------------------------------------------------------
+
+namespace SampleClaimsProvider
+{
+    using Microsoft.AspNetCore.Mvc;
+
+    [Route("api/healthcheck")]
```

Comment:
Is this a CVP required route?
If so, we should document the function below to say that it just needs to return a 200 and whether or not body content is required.